### PR TITLE
Develop.website rebuild

### PR DESCRIPTION
**Pull request by GerbenBosschieter**

### CODE REVIEW

File: src/sources/forus-platform/pug/tpl/directives/website/navbar.pug [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+              a(href='#') Page 1-3
+        li
+          a(ui-sref="me") Me App
+        li
+          a(ui-sref="current") Over ons
+        li 
+          a(ui-sref="contact") Contact
+      ul.nav.navbar-nav.navbar-right
+        li
+          a.login_btn(ng-click="openAuthPopup()")(ng-if="!$root.auth_user")
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/directives/website/navbar.pug]: /var/www/forus-frontend/src/sources/forus-platform/pug/tpl/directives/website/navbar.pug, line 31:
You should not have pug tags with multiple attributes.

)( should be removed

File: src/sources/forus-platform/pug/layout/includes/scripts.pug [hunk 2]
```diff
@@ -4,4 +4,6 @@
 script(src=src_prefix + "./assets/dist/bundle/js/bundle.min.js")
 
 //- custom script
-script(src=src_prefix + "./assets/js/app.min.js")
\ No newline at end of file
+script(src=src_prefix + "./assets/js/app.min.js")
+
+<script type="text/javascript" > window.ZohoHCAsap=window.ZohoHCAsap||function(a,b){ZohoHCAsap[a]=b;};(function(){var d=document; var s=d.createElement("script");s.type="text/javascript";s.defer=true; s.src="https://desk.zoho.eu/portal/api/web/inapp/15870000001702866?orgId=20065804523"; d.getElementsByTagName("head")[0].appendChild(s); })(); </script>
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/layout/includes/scripts.pug]:
```js
script(type="text/javascript").
   window.ZohoHCAsap=window.ZohoHCAsap||fu.....
```

File: src/sources/forus-platform/pug/tpl/directives/website/footer.pug [hunk 3]
```diff
@@ -1,252 +1,40 @@
...
+      .col-6.col-md-4.contact
+        h4 Vragen of samenwerking ?
+        p Heb je vragen of wil je met ons samenwerken? Neem dan contact met ons op.
+        a.main_button(href="#" data-toggle="modal" data-target="#modalContact") Direct contact
+.col.sub-foot
+  .container
+    a.footer-brand(href="/#!/")
+      include ../../../../resources/platform-general/assets/img/svg/forus-logo-footer.svg
+    ul.pull-right
+      li
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/directives/website/footer.pug]: https://nl.linkedin.com/company/stichtingforus

File: src/sources/forus-platform/pug/tpl/directives/website/footer.pug [hunk 4]
```diff
@@ -1,252 +1,40 @@
...
+        include ../../../../resources/platform-general/assets/img/svg/linkedin.svg
+      li
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/directives/website/footer.pug]: https://chat.forus.io

File: src/sources/forus-platform/pug/tpl/directives/website/footer.pug [hunk 5]
```diff
@@ -1,252 +1,40 @@
...
+        include ../../../../resources/platform-general/assets/img/svg/rocketchat.svg
+      li
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/directives/website/footer.pug]: https://github.com/teamforus

**Comment by maxvisser** [comment]
I know its not ready for review yet but login success model is even more crap then before:
image

maybe we can make a better one in figma and fix them everywhere (landing pages, webshop etc)

File: src/sources/forus-platform/pug/tpl/directives/website/footer.pug [hunk 6]
```diff
@@ -1,252 +1,48 @@
...
-                        //li
-                            a(href="#") Privacy & Terms
-
-                        //li
-                            a(href="#") Contact Us
-
+      .col-6.col-md-4
+        h5 Contactgegevens
+        ul
+          li: p E-mailadres: <span><a href="mailto:info@forus.io">info@forus.io</a></span>
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/directives/website/footer.pug]: does this not work?
```pug
li: p 
      | E-mailadres: 
      span
         a(href="mailto:info@forus.io")
         | support@forus.io
```

Comment (GerbenBosschieter) [reply src/sources/forus-platform/pug/tpl/directives/website/footer.pug]: Well check it out locally if you want to know

**Review by maxvisser — APPROVED** [review]

**Pull request merged**